### ISSUE TITLE
upgrade ink to version with sr25519

### ISSUE THREAD

**Issue by goastler** [prosopo/captcha#367]

**Comment by goastler** [comment]
waiting on an ink release >4.2.1 which was when my ink sr25519 PR was merged. Thus it will be present in whatever version comes after 4.2.1

**Comment by HughParry** [comment]
New ink versions now out. Moving to low priority since we've worked without it for a long time and it got unblocked on August 24th https://github.com/use-ink/ink/releases
